app/scripts/lib/experiment.js

```diff
@@ -151,6 +151,9 @@ define(function (require, exports, module) {
         if (initResult) {
           this._activeExperiments[experimentName] = experiment;
           this.metrics.logExperiment(experimentName, experiment._groupType);
+          this.notifier.trigger('flow.event', {
```

Comment:
Could `metrics.logExperiment` just invoke it's own `logFlowEvent`?

In addition, do you only want to log this once? If I keep switching between signin and signup, multiple experiment events are logged:

<img width="712" alt="screen shot 2017-03-02 at 13 24 06" src="https://cloud.githubusercontent.com/assets/848085/23508911/a421f8c2-ff4b-11e6-9f35-e58ef607c33b.png">